src/core_plugins/kibana/public/discover/index.html

```diff
@@ -28,6 +28,7 @@ <h1 tabindex="0" id="kui_local_breadcrumb" class="kuiLocalBreadcrumb">
           query="state.query"
           app-name="'discover'"
           on-submit="updateQueryAndFetch($query)"
+          index-pattern="indexPattern"
```

Comment:
We need to make the same change on visualize and dashboard. What should happen on dashboards where there could be 0 or more index patterns? 

We also probably need to handle the case where no index pattern is passed more gracefully, if we want to support that use case, because right now it spams the console with errors.